cmd/kubeadm/app/preflight/checks.go

```diff
@@ -435,15 +426,20 @@ func (sysver SystemVerificationCheck) Check() (warnings, errors []error) {
 	// Run the system verification check, but write to out buffered writer instead of stdout
 	bufw := bufio.NewWriterSize(os.Stdout, 1*1024*1024)
 	reporter := &system.StreamReporter{WriteStream: bufw}
+	runtimeOS := runtime.GOOS
 
 	var errs []error
 	var warns []error
-	// All the validators we'd like to run:
+	// All the common validators we'd like to run:
 	var validators = []system.Validator{
-		&system.OSValidator{Reporter: reporter},
 		&system.KernelValidator{Reporter: reporter},
-		&system.CgroupsValidator{Reporter: reporter},
-		&system.DockerValidator{Reporter: reporter},
+		&system.DockerValidator{Reporter: reporter}}
+
+	if runtimeOS != "windows" {
```

Comment:
Can we switch this to `runtime.GOOS == "linux"` instead?

Reply:
fixed